chapters/ch02.md

```diff
@@ -1,38 +1,31 @@
 # Why Machine Learning Strategy
```

Comment:
Tiều đề chương này copy từ bản gốc bị sai. Đúng phải là "How to use this book to help your team"
Cậu sửa lại nhé.